conf/ocsci/odf_cli_dr.yaml

```
@@ -0,0 +1,95 @@
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)